packages/mui-material-nextjs/src/v14-appRouter/index.ts

```diff
@@ -1,2 +1 @@
-'use client';
```

Comment:
Only "use client" where it's needed, there is no client-side React API used here.